app/controllers/concerns/new_inprocess_controller.rb

```diff
@@ -8,10 +8,18 @@ def index
     order_details = new_or_in_process_orders.joins(:order)
 
     @search_form = TransactionSearch::SearchForm.new(params[:search], defaults: { date_range_field: "ordered_at", allowed_date_fields: ["ordered_at"] })
-    @search = TransactionSearch::Searcher.new(TransactionSearch::ProductSearcher,
-                                              TransactionSearch::OrderedForSearcher,
-                                              TransactionSearch::OrderStatusSearcher,
-                                              TransactionSearch::DateRangeSearcher).search(order_details, @search_form)
+    searchers = [
+      TransactionSearch::ProductSearcher,
+      TransactionSearch::OrderedForSearcher,
+      TransactionSearch::OrderStatusSearcher,
+      TransactionSearch::DateRangeSearcher,
+    ]
+
+    if SettingsHelper.feature_on?(:cross_core_order_view)
+      searchers.insert(2, TransactionSearch::CrossCoreSearcher)
```

Comment:
View heavily relies on the searchers order, that's why I'm adding it in the third position

Reply:
Leave it at the end and make column smaller for this input